iothub/service/src/AmqpServiceClient.cs

```diff
@@ -159,12 +159,6 @@ public async override Task SendAsync(string deviceId, Message message, TimeSpan?
             }
         }
 
-        // This call is executed over HTTP.
```

Comment:
What are the implications of deleting these methods? How is it being referenced?

Reply:
No implications of deleting these, the same functionality is still available through the overload which takes in an optional cancellation token

Reply:
Got it.